src/geo/transform.js

```diff
@@ -1639,6 +1683,20 @@ class Transform {
         this.worldToFogMatrix = this._camera.getWorldToCameraPosition(cameraWorldSize, cameraPixelsPerMeter, windowScaleFactor);
     }
 
+    _calcDistanceMatrices() {
```

Comment:
If this is only used for debug, how could we only run the code below only when it's necessary? Feels like a code path most common use cases wouldn't need.